packages/robotcode/package.yaml

```
@@ -0,0 +1,17 @@
+---
+name: robotcode
+description: The Ultimate Robot Framework Toolset
+homepage: https://robotcode.io/
+licenses:
+  - Apache-2.0
+languages:
+  - Robot Framework
+categories:
+  - LSP
+  - DAP
+
+source:
+  id: pkg:pypi/robotcode[all]@1.0.3
```

```suggestion
  id: pkg:pypi/robotcode[all]@1.0.3


schemas:
  lsp: vscode:https://raw.githubusercontent.com/robotcodedev/robotcode/v{{version}}/package.json
```

We should add the schema for the LSP configuration